test/integration/servers/trigger.spec.ts

```diff
@@ -48,18 +48,18 @@ let userType3: UserRecord
 beforeAll(async () => {
   await bitcoindClient.loadWallet({ filename: "outside" })
 
-  await createUserWallet(0)
-  await createUserWallet(3)
-  await createUserWallet(12)
+  await createUserWalletFromUserRef("A")
+  await createUserWalletFromUserRef("D")
+  await createUserWalletFromUserRef("F")
 
-  walletId0 = await getDefaultWalletIdByTestUserIndex(0)
-  walletId3 = await getDefaultWalletIdByTestUserIndex(3)
-  walletId12 = await getDefaultWalletIdByTestUserIndex(12)
+  walletId0 = await getDefaultWalletIdByTestUserRef("A")
+  walletId3 = await getDefaultWalletIdByTestUserRef("D")
```

Comment:
Now that we are using references, names with indexes like `walletId3` no longer make sense

Reply:
fixed